pkg/cmd/operator/mustgather.go

```diff
@@ -192,6 +192,14 @@ var mustGatherSpecs = []struct {
 		Namespace: corev1.NamespaceAll,
 		Name:      "scylla-manager",
 	},
+	{
+		GroupResource: schema.GroupResource{
+			Resource: "namespaces",
+			Group:    "",
+		},
+		Namespace: corev1.NamespaceAll,
+		Name:      "scylla-operator-node-tuning",
```

Comment:
does this need to be conditional as the namespace is not there in some configurations?

Reply:
no, collector ignores when resource we search for is not there:
https://github.com/scylladb/scylla-operator/blob/6c7657d58b464a013e04cb6abe31dd6a8fdaf539/pkg/gather/collect/collect.go#L468-L470